tests/components/local_todo/conftest.py

```diff
@@ -45,6 +49,8 @@ def _mock_exists(self) -> bool:
         return self._content is not None
 
     def _mock_read_text(self) -> str:
+        if self._raise_error:
```

Comment:
Can we use a `Mock` here instead and accept a side effect instead which we always set on the mock. By default the side effect could be `None` and the return value as on line 54.